plugins/woocommerce-blocks/assets/js/blocks/cart-checkout-shared/block-settings/index.tsx

```diff
@@ -44,130 +35,4 @@ export const BlockSettings = ( {
 	);
 };
 
-const ExpressPaymentButtonStyleControls = ( {
```

Comment:
All the express payment controls were moved to the file above `express-payment-settings.tsx`